### DISCUSSION TITLE
Tutorial on building block-based templates in classic themes

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#109] [Topic Ideas]
I've written an [existing post](https://gutenbergtimes.com/building-a-block-based-header-template-in-a-classic-theme/) on building a block-based header template in a classic theme for Gutenberg Times. We could look into tweaking/updating that post and porting it over.

Alternatively, we could explore some other examples. One that was passed along to me was the need to add a customizable block (cookie banner block in this instance) to a classic theme.

Regardless of the content, it'd be good to have a walk-through of using more blocks within classic themes on the Dev Blog.

**Comment by justintadlock** [comment]
We can probably also explore placing something like the cookie banner (or similar things where user placement is less needed) on a broader scale where it applies to more themes.

**Comment by bph** [comment]
Created issue, please continued discussion there. 

- https://github.com/WordPress/developer-blog-content/issues/116

This discussion is locked.
